boards/nordic/nrf54l15dk/nrf54l15dk_nrf54l05_cpuapp.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf54l15dk/nrf54l15/cpuapp
```

Comment:
Shouldn't be "nrf54l15dk/nrf54l05/cpuapp"?
And name change below?

Reply:
fixed, thanks!